util/project_allocation_mapper.py

```diff
@@ -237,6 +255,26 @@ def update_project_nickname(project_id, project_charge_code, nickname):
             pextras.nickname = nickname
             pextras.save()
 
+    def update_user_profile(self, user, new_profile, is_request_pi_eligibililty):
+        new_profile['piEligibility'] = user['piEligibility']
+        if is_request_pi_eligibililty:
+            if self.is_from_db:
+                pie_request = PIEligibility()
+                pie_request.requestor_id = user['id']
+                pie_request.save()
+            else:
+                self.tas.save_user(user['id'], {'piEligibility': 'Requested'})
+            self._create_ticket_for_pi_request(user)
+
+        keycloak_client = KeycloakClient()
+        keycloak_client.update_user(user['username'],
+                                    affiliation_title=new_profile['title'] if 'title' in new_profile else None,
```

Comment:
```suggestion
                                    affiliation_title=new_profile.get('title'),
```
I think that is equivalent?